config/src/main/java/com/yahoo/config/subscription/impl/JRTConfigRequester.java

```diff
@@ -234,6 +234,7 @@ private void handleOKRequest(JRTClientConfigRequest jrtReq,
         noApplicationWarningLogged = Instant.MIN;
         connection.setSuccess();
         sub.setLastCallBackOKTS(Instant.now());
+        log.log(LogLevel.DEBUG, "OK response received in handleOkRequest: " + jrtReq);
```

Comment:
Consider using a lambda instead to ensure toString is only invoked if debug-logging is enabled.
```log.log(LogLevel.DEBUG, () -> "OK response received in handleOkRequest: " + jrtReq);```

Reply:
Yes, thanks, forgot that this time, fixed